r/BANEScarparkinglite/LICENSE

```diff
@@ -1,7 +1,2 @@
-Copyright 2017 © Owen Jones (olj23@bath.ac.uk)
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
Does this license change affect the license Bath-ML/parking must have? I don't really understand what the new license entails

Reply:
I wasn't completely clear on this myself. CRAN are very particular about how the LICENSE file should be formatted - it must only contain a name and year. The package as a whole is still licensed under MIT, as specified in the DESCRIPTION file.

Reply:
That's odd. Can you link me to the CRAN license rules please?

Reply:
Having investigated a little further:

* In the package's DESCRIPTION file, I have specified that I want to use the MIT license with the line `License: MIT`
* The MIT "license" this refers to is actually a license _template_ - it looks like this: https://www.r-project.org/Licenses/MIT
* So I need to specify `License: MIT + file LICENSE` in the DESCRIPTION file, and in that file I need to specify how to fill in the template

So the license is populated when the package is built... but there's no full copy of the license in the GitHub repo.

This issue has (of course) been discussed in some detail for other better-known packages: https://github.com/tidyverse/tidyr/pull/267

The favoured solution appears to be to add a separate `LICENSE.md` file containing the full license text, so this is what I have done in the latest commit.